.github/workflows/release-1-branch.yml

```diff
@@ -0,0 +1,79 @@
+name: 'CD: branch'
+
+on:
+  schedule:
+    # It's good to deploy early in week:
+    # - In the best case, will QA, tag and deploy on the same Tuesday or Wednesday.
+    # - In an ok case, will QA, tag and deploy on the next Monday-Wednesday.
+    # - In the worst case, the last chance is to deploy on after-next Monday.
+    - cron: '17 5 * * 2'
```

Comment:
If I read this correctly, it's every Tueday at 5:17 UTC. 

I'm curious why '17 , not 15 or 30?